spec/units/user_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'user'
+require 'database_helpers'
+
+describe User do
+  context '#create' do
+    it 'creates a new user' do
+      user = User.create(name: 'John', email: 'test@email.com', password: 'test-password')
+      persisted_data = persisted_data(table: :users, id: user.id)
+      expect(user).to be_a User
+      expect(user.id).to eq persisted_data.first['id']
```

Comment:
I wonder whether we need 'persisted_data' as testing element. It is included in bookmarks challenge as a way to test the id of the first bookmark entry. Let me know your thoughts - I've been reading this section to try and get my head round it: https://github.com/makersacademy/course/blob/main/bookmark_manager/walkthroughs/11.md - section is 'returning a bookmark from bookmarks.create (rather than bookmarks.all)